core/node/events/snapshot.go

```diff
@@ -903,14 +915,18 @@ func findUserMembership(
 	memberships []*UserPayload_UserMembership,
 	streamId []byte,
 ) (*UserPayload_UserMembership, error) {
-	return findSorted(
+	md, err := findSorted(
 		memberships,
 		streamId,
 		bytes.Compare,
 		func(membership *UserPayload_UserMembership) []byte {
 			return membership.StreamId
 		},
 	)
+	if err != nil {
+		return nil, AsRiverError(err).Func("findUserMembership")
+	}
+	return md, nil
```

Comment:
thanks for adding these errors - as far as i know, these errors should just be ignored and not cause any disruption in the snapshotting flow.